ci/util/files.go

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/go-openvpn" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package util
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var excludedDirs = []string{".git", "vendor"}
```

Comment:
We are using the same `excludedDirs` in several places, maybe it should be one place?